packages/xdl/src/Versions.js

```diff
@@ -162,7 +157,7 @@ export async function canTurtleBuildSdkVersion(sdkVersion, platform) {
     );
   }
 
-  const turtleSdkVersions = await turtleSdkVersionsAsync();
+  const supportedVersions = await getSdkVersionsSupportedByTurtle();
   const expoSdkVersion = (await sdkVersionsAsync())[sdkVersion];
```

Comment:
I think this part might not be required anymore? It fetches all the SDK versions to check if the specified version is valid and not deprecated. But if the version is included in the list returned by `standalone-build/supportedSDKVersions`, it should be valid and not deprecated?

Reply:
You're right, I changed that. Thanks!